content/en/docs/_index.md

```diff
@@ -5,14 +5,18 @@ menu: { main: { weight: 10 } }
 aliases: [/docs/workshop/*]
 ---
 
-OpenTelemetry, also known as OTel for short, is a vendor-neutral open source
+OpenTelemetry, also known as OTel, is a vendor-neutral, open-source
```

Comment:
FYI, since I also learned this a while back: "open source" and "cloud native" without a hyphen is one of the few things the CNCF style guide dictates: https://github.com/cncf/foundation/blob/main/style-guide.md#1-cloud-native-and-open-source

Reply:
Interesting. We seem to contradict existing style guides in that sense — I was about to open a separate PR to remove the rule: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source

When used as adjective, it should bear a hyphen... But I'll just disagree and commit :P